elements/rh-footer/rh-footer.css

```diff
@@ -27,6 +27,7 @@ footer,
 
 ::slotted(:is(h1,h2,h3,h4,h5,h6)) {
   font-family: var(--rh-font-family-heading, RedHatDisplay, "Red Hat Display", "Noto Sans Arabic", "Noto Sans Hebrew", "Noto Sans JP", "Noto Sans KR", "Noto Sans Malayalam", "Noto Sans SC", "Noto Sans TC", "Noto Sans Thai", Helvetica, Arial, sans-serif);
+  line-height: var(--rh-line-height-heading, 1.3);
```

Comment:
We also want to add this to the shadowroot as well in case this element is located outside the scope of the lightdomcss.